src/androidTest/java/com/owncloud/android/lib/resources/files/SearchRemoteOperationIT.java

```diff
@@ -306,6 +306,37 @@ public void testPhotoSearchTimestamps() throws IOException {
         assertEquals(5, result.getResultData().size());
     }
 
+    @Test
+    public void testPhotoSearchLimitDates() throws IOException {
+        for (int i = 0; i < 10; i++) {
+            String filePath = createFile("image" + i);
+            String remotePath = "/image" + i + ".jpg";
+            assertTrue(new UploadFileRemoteOperation(
+                    filePath,
+                    remotePath,
+                    "image/jpg",
+                    String.valueOf(1464818400 + i))
```

Comment:
What does this number mean? Maybe best to make it a constant so it has a name at least

Reply:
It is a random unix time. I will adjust it :)